js/web/lib/wasm/jsep/webgpu/ops/unary-op.ts

```diff
@@ -29,7 +30,11 @@ const createElementwiseProgramShader =
       const output = outputVariable('outputData', outputDataType, [vecSize], 4);
 
       return `
-      ${shaderHelper.registerUniform('vec_size', 'u32').declareVariables(input, output)}
+      ${
+          (additionalUniformsType ?
```

Comment:
can just use 
```
const arr = [ { .. } ];
if (additionalUniformsType) {
  arr.push(...additionalUniformsType);
}
```
to avoid code duplicate.

same to additionalUniforms to below

Reply:
Done